src/app/components/Sidebar/Sidebar.jsx

```diff
@@ -3,8 +3,20 @@ import PropTypes from 'prop-types';
 import { LeftWedgeIcon } from '@nypl/dgx-svg-icons';
 
 import BookFilters from '../BookFilters/BookFilters.jsx';
+import ListFilter from '../ListFilter/ListFilter.jsx';
 import config from '../../../../appConfig';
 
+// This data is a temporary dummy data for creating the season list
+const fieldsetProps = {
```

Comment:
The dummy data of the season list. It will come from the API (I assume) in the future PRs.